cloud/deployment/deployment_variable.go

```diff
@@ -218,6 +221,54 @@ func addVariableFromFlag(oldKeyList []string, oldEnvironmentVariables []astro.En
 	return newEnvironmentVariables
 }
 
+func addVariablesFromArgs(oldKeyList []string, oldEnvironmentVariables []astro.EnvironmentVariablesObject, newEnvironmentVariables []astro.EnvironmentVariable, variableList []string, updateVars, makeSecret bool) []astro.EnvironmentVariable {
+	var key string
+	var val string
+	// valisdate each key value pair and add to new variables list
+	for i := range variableList {
+		// split pair
+		pair := strings.Split(variableList[i], "=")
+		if len(pair) > 1 {
+			key = pair[0]
+			val = pair[1]
+			if key == "" || val == "" {
+				fmt.Printf("Input %s has blank key or value\n", variableList[i])
+				continue
+			}
+		} else {
+			fmt.Printf("Input %s is not a valid key value pair\n", variableList[i])
```

Comment:
might want to mention the correct format in the error response
```suggestion
			fmt.Printf("Input %s is not a valid key value pair, should be of the form key=value\n", variableList[i])
```